testdata/expected/report/sarif_simple.sarif

```diff
@@ -169,6 +169,20 @@
         "text": "SG\\.[\\w_]{16,32}\\.[\\w_]{16,64}"
        }
       },
+      {
```

Comment:
NOTE1: Since I added the tests to detect_test with `cfgName: "simple",`, I realized then I had to add my new rules to the simple.toml file. And then I realized I had to add them to this expected/report/sarif_simple.sarif file.

So my detect_test changes ended up spreading into multiple other files...let me know if there was a better way to do this. Or if it's better to just revert my changes to detect_test.

NOTE2: I pretty much just copied these expected results from the test failure output 🤷